pymongo/collection.py

```diff
@@ -1379,6 +1379,8 @@ def create_indexes(self, indexes):
            Apply this collection's write concern automatically to this operation
            when connected to MongoDB >= 3.4.
         .. versionadded:: 3.0
+
+        .. _createIndexes: https://docs.mongodb.com/manual/indexes/
```

Comment:
Let's use dochub links instead. They are guaranteed to resolve to the right thing seemingly forever.

http://dochub.mongodb.org/core/indexes